tests/test_models/test_forward.py

```diff
@@ -815,7 +815,7 @@ def test_maskformer_forward():
 
 def test_mask2former_forward():
     model_cfg = _get_detector_cfg(
-        'mask2former/mask2former_r50_lsj_8x2_50e_coco.py')
+        'mask2former/mask2former_r50_lsj_8x2_50e_coco_pan.py')
```

Comment:
Need add unit test for instance segmentation

Reply:
Reworked the unit test to test both configs.  Added tests when gt_semantic_seg=None and a test to check the output type (pan is dict, ins is tuple).